Tests/test_inventory.gd

```diff
@@ -28,11 +28,13 @@ func test_add_relic() -> void:
 
 func test_remove_relic() -> void: 
 	var test_relic : Relic = load("res://Items/test_relic.tres")
+	var test_relic_2 : Relic = load("res://Items/test_relic_2.tres")
 	InventoryManager.relic_component.add_relic(test_relic)
+	InventoryManager.relic_component.add_relic(test_relic_2)
 	InventoryManager.relic_component.add_relic(test_relic)
-	InventoryManager.relic_component.add_relic(test_relic)
+	InventoryManager.relic_component.lose_relic(test_relic_2)
 	InventoryManager.relic_component.lose_relic(test_relic)
-	assert_eq(InventoryManager.relic_component.get_held_relics(), [test_relic, test_relic])
+	assert_eq(InventoryManager.relic_component.get_held_relics(), [test_relic])
```

Comment:
I think you misunderstood what i said about losing relics. Basically, you should split this test in two, in both tests you start by adding the two relics.
In the first test you lose the the relic 1 and ensure the list is just [relic_2]
In the second test, you lose the relic 2 and ensure the list is just [relic_1]
This is not equivalent to what you did there, since again we don't really see the order of the stuff